arches/install/arches-templates/package.json

```diff
@@ -20,6 +22,9 @@
     "dependencies": {
         "arches": "archesproject/arches#{{ arches_version }}"
     },
+    "resolutions": {
+        "@vue/compiler-sfc": "^3.4.21"
+    },
```

Comment:
does this do anything? I'm reticent to include it at the project level if it's just quieting noise

Reply:
This has been resolved in #10980